server/metadata_queue.go

```diff
@@ -0,0 +1,23 @@
+package server
+
+import (
+	"context"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/livepeer-data/pkg/event"
+)
+
+const queuePublishTimeout = 1 * time.Second
+
+var MetadataQueue event.Producer
+
+func BackgroundPublish(queue event.Producer, key string, body interface{}) {
```

Comment:
Can this be a method on the producer?

Reply:
I guess that makes sense. I think I will make it a helper in the `event` lib instead which takes the producer itslef as an argument. Otherwise every `Producer interface` implementation would have to rewrite it, when it only needs the public interface to work.

Reply:
Decided to get rid of this helper altogether.